### DISCUSSION TITLE
How can I get provisional profile expiry date

### DISCUSSION THREAD

**Discussion by hanumanAbacus** [fastlane/fastlane#20343] [Q&A] [answered]
I need to get the provisional profile expiry date used when building the app using gym

**Comment by crazymanish** [comment, marked as answer]
- Step1: Find the provisional profile file(s) - [see steps here](https://stackoverflow.com/a/52404463/3778686)
- Step2: Open those files for the expiry date

i hope this will help you!

**Comment by hanumanAbacus** [comment]
@crazymanish I understand, do you think we can get it using fastlane

**Reply by crazymanish** [reply]
i never need it but it should be possible using Fastlane too...at-least, i can see it inside the spaceship model https://github.com/fastlane/fastlane/blob/e874a47c6e2e0e61590a03d3b71e75e5a505d1ce/spaceship/lib/spaceship/portal/provisioning_profile.rb#L30
